google_guest_agent/instance_setup.go

```diff
@@ -75,6 +90,142 @@ func agentInit() error {
 			}
 			break
 		}
+	} else {
+		// Check if instance ID has changed, and if so, consider this
+		// the first boot of the instance.
+		instance_id, err := ioutil.ReadFile("/etc/instance_id")
```

Comment:
Can you add a TODO in the Windows comment above to make this section cross platform. There's a number of one time instance setup things I'd like to remove from instance_setup.ps1